includes/admin/class-wcmp-export.php

```diff
@@ -1131,25 +1131,32 @@ public static function getItemWeight_kg($item, WC_Order $order): float
         }
 
         $weight      = (int) $product->get_weight();
+        $product_weight = WCMP_Export::calculatedKiloWeight($weight);
+
+        $item_weight = (float) $product_weight * (int) $item["qty"];
+
+        return (float) $item_weight;
+    }
+
+    public static function calculatedKiloWeight($weight): float
+    {
         $weight_unit = get_option("woocommerce_weight_unit");
```

Comment:
misschien kun je deze meteen meenemen en gewoon camelCasen

Reply:
Dit is een method van WordPress

Reply:
`$weight_unit` -> `$weightUnit` denk ik dat er wordt bedoeld

Reply:
precies

Reply:
Deze had ik al aangepast, ik denk dat je moet kijken naar alle changes